packages/jsdoc-plugins/lib/relation-fixer/addmissingdoclets.js

```diff
@@ -33,12 +37,26 @@ function addMissingDoclets( originalDoclets ) {
 		}
 	}
 
-	const entitiesWhichNeedNewDoclets = clonedDoclets.filter( d => {
-		return d.kind === 'class' || d.kind === 'interface' || d.kind === 'mixin';
+	extendTypedefs( typedefDoclets );
```

Comment:
Side effect. You should pass the `typedefDoclets` object and expect that this function will return a new object.

Reply:
This part of the code is full of side effect. But I don't want to change it within that PR.